master/internal/stream/system.go

```diff
@@ -2,20 +2,23 @@ package stream
 
 import (
 	"context"
-	"time"
-	"fmt"
 	"encoding/json"
+	"fmt"
+	"time"
 
 	"github.com/gorilla/websocket"
 	"github.com/labstack/echo/v4"
-	"github.com/pkg/errors"
 	"github.com/lib/pq"
+	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 
+	detContext "github.com/determined-ai/determined/master/internal/context"
+	"github.com/determined-ai/determined/master/pkg/model"
 	"github.com/determined-ai/determined/master/pkg/stream"
 )
 
 // JsonB is the golang equivalent of the postgres jsonb column type.
+// nolint
```

Comment:
I'm curious, what does the linter have to say about this type?

Reply:
It really doesn't like the name JsonB (it wants it to be JSONB)

Reply:
so just rename it then?